x-pack/plugins/security_solution/public/resolver/store/data/action.ts

```diff
@@ -14,9 +15,9 @@ interface ServerReturnedResolverData {
      */
     result: ResolverTree;
     /**
-     * The database document ID that was used to fetch the resolver tree
+     * The database parameters that was used to fetch the resolver tree
      */
-    databaseDocumentID: string;
+    parameters: TreeFetcherParameters;
```

Comment:
instead of keeping track of just the `databaseDocumentID` used in the request, we now also keep track of the `indices`.